api/app/controllers/api/v1/applications_controller.rb

```diff
@@ -47,6 +54,22 @@ def update
             end
         end
 
+        # /applications/delete or /applications/:id/delete
```

Comment:
Again, if `id` is required, we should be never able to access `applications/delete` right?